shotgun_api3/shotgun.py

```diff
@@ -469,7 +470,8 @@ def __init__(self,
                  password=None,
                  sudo_as_login=None,
                  session_token=None,
-                 auth_token=None):
+                 auth_token=None,
+                 localized=True):
```

Comment:
I might default this to `False`. My fear would be that this will change the form of the data returned by the API and might break existing client scripts (or even stuff in Toolkit that we provide) without additional code changes to compensate for it.